ui/test/nuxeo-video-conversions.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, flush, html, isElementVisible } from '@nuxeo/nuxeo-elements/test/test-helpers';
+import '../nuxeo-video/nuxeo-video-conversions.js';
+import videoProperties from './resources/videoProperties';
+/* eslint-disable no-unused-expressions */
+
+suite('nuxeo-video-conversions', () => {
+  suite('When no document provided', () => {
+    test('Should still display the label', async () => {
```

Comment:
We should be aligning our test description with BDD style, with `should + when` formulation.

Reply:
Indeed, we are trying to use, as much as possible, that convention for unit test naming: `Should ExpectedBehavior When StateUnderTest`.
Anyway, there is always a level of subjectivity and creativity here, which is good 🙂 

Do you think it would make sense to apply this convention on these tests, @andreacornaglia ?

Reply:
if we're trying to use BDD, why are we not using describe and it ?

Reply:
BDD API for ChaiJS and TDD API for Mocha, I guess.